ui-tests/src/test/java/io/syndesis/qe/steps/other/ConcurValidationSteps.java

```diff
@@ -0,0 +1,86 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.utils.HttpUtils;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.FormBody;
+import okhttp3.Headers;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+/**
+ * Maybe this will become handy in the future when we receive correct credentials for both
+ * OAuth and WS and both will point to the same instance. For now this class is not used.
+ */
+@Slf4j
+public class ConcurValidationSteps {
+    private static String ACCESS_TOKEN = null;
+    private static final String ACCESS_TOKEN_ENDPOINT = "https://us-impl.api.concursolutions.com/oauth2/v0/token";
+    // you can find which endpoint you need and how to use it here:
+    // https://developer.concur.com/api-reference/expense/expense-report/reports.html
+    private static final String CONCUR_ENDPOINT_EXPENSE_REPORTS = "https://implementation.concursolutions.com/api/v3.0/expense/reports";
+
+    @When("^check that concur list or whatever we need to test is here \"([^\"]*)\"$")
+    public void invokeRequest(String something) {
+
+        String content = "{\"name\": \"report-from-testsuite\"}";
+
+        Response res = HttpUtils.doPostRequest(
+                CONCUR_ENDPOINT_EXPENSE_REPORTS,
+                content,
+                "application/json",
+                Headers.of("Authorization", "Bearer " + getAccessToken(), "Accept", "application/json")
+        );
+        log.info("Response code: " + res.code());
+        log.info(res.toString());
+    }
+
+    /**
+     * Prepare access token valid access token
+     */
+    private String getAccessToken() {
+        if (ACCESS_TOKEN != null) {
+            return ACCESS_TOKEN;
+        }
+
+        log.info("Generating access token from concur API....");
+
+        FormBody.Builder formBuilder = new FormBody.Builder()
+                .add("grant_type", "password")
+                .add("credtype", "password");
+
+        //TODO: this needs to be loaded from credentials.json via accounts
```

Comment:
why is this todo?

Reply:
When I found out that I can not use those validation steps due to concur giving us two different credentials pointing to two different instances of concur, I stopped development of this class. But it works and may be used in the future.